decoders/connector/mokosmart/LW008-MT/connector.jsonc

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Thank you for opening a Pull Request to add a public decoder. We appreciate your effort and contribution to the TagoIO IoT community.

Don't need to open a Pull Request for each file, you can combine all the files related to the sensor decoder into a single Pull Request.

Watch this video to guide you: https://www.youtube.com/watch?v=7ejN2q0YWo0